### PR TITLE
[WIP] "Futurizes" a plethora of firearms and ammo to be more in-line with the setting

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

As said in the PR title - This aims to replace a bunch of absolutely outdated ammo and certain guns to be more in-line with our setting. We've been trying to move away from the retrofuture thing forever & this is another step in the right way

## How This Contributes To The Skyrat Roleplay Experience

This makes the setting more believable while not interfering with gameplay in any major ways (and I'd doubt even minor ways). Except for some names and aesthetics, everything stays mostly the same but it has a sprinkle of 26th century on it.

## Changelog

I'll complete this when I'm done with everything

:cl:
qol: Certain unspent ammo casings have bigger, full sized sprites for ease of clicking on.
imageadd: Added unique casing sprites for 5.45
imageadd: Resprited AKM and Modern AKM
tweak: AKM and Modern AKM renamed to "Krinkov" and "Bubba's Krinkov". They're now chambered in 5.45x39mm 7N30, and are statistically the same weapons.
tweak: M16, Modern M16 and M16 Patriot renamed to "XM-2537", "XM-2537 SOPMOD" and "XM-2537 Amans Patriae" respectively.
spellcheck: 7.62, .50 BMG, .50 AE, 5.56, .45, 4.6x30, 7.62x39 M43 renamed to 5.8x42mm Mk. VII, .416 Stabilis, .454 Trucidator, .277 Aestus, .460 Ceres, 6.5x25mm and 5.45x39mm 7N30-B respectively.
spellcheck: AK projectile no longer incorrectly says 7.62x25mm
fix: XM2537 (M16) magazines no longer incorrectly hold 20 rounds instead of 30
tweak:
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
image
Byebye. You see? Byebye. You see? You see? U C? Noooo more 20th century ammo! No more! No more. No more 20th century ammo.

**Comment by softcerv** [comment]
Rip calling Mosins, Mosins

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 1]
```diff
@@ -301,13 +301,13 @@
...
 	name = "\improper NT CMG-1"
-	desc = "A bullpup three-round burst .45 PDW with an eerily familiar design. It has a foldable stock and a dot sight."
+	desc = "A bullpup three-round burst .460 PDW with an eerily familiar design, designed for espionage operations. It has a foldable stock and a dot sight."
 	icon_state = "cmg1"
 	icon = 'modular_skyrat/modules/aesthetics/guns/icons/guns.dmi'
 
 /obj/item/gun/ballistic/automatic/ar/modular/model75
 	name = "\improper NT ARG-75"
-	desc = "A contemporary rifle that just exited its prototype phase and has started production, a rare sight for now. It's equipped with a heavy duty integrally suppressed barrel, 1.4 times scope and a topmounted laser sight."
+	desc = "A contemporary rifle that's fresh out of its prototype phase and has in production, a rare sight for now. It's equipped with a heavy duty integrally suppressed barrel, 1.4 times scope and a topmounted laser sight."
```
Comment (Draganfrukts) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm]:
```suggestion
	desc = "A contemporary rifle that's fresh out of its prototype phase and is currently in production, a rare sight for the time being. It's equipped with a heavy duty integrally suppressed barrel, 1.4 times scope and a topmounted laser sight."
```

**Comment by KathrinBailey** [comment]
spongebob-who-asked
image
People are going to see this and think "oh arma 3 MX bullet, it must be the same or a bit higher damage than 5.56 and go in rifles" when in reality it's the crappy WT550 bullet.

image
People are going to think the... 50 action express..? is worse than the renamed .45 with this one.

image
No one outside of Texas has a clue what this means and won't know it's supposed to be armor-piercing.

image
It's literally just a M16. If you don't want 20th century firearms in the game, remove them and replace them with something new. This is incredibly lazy, you can easily change the typepath from `automatic/m16` to `automatic/xm2537` without causing issues since this is modular.

Everyone who has played a shooter game knows what `9mm`, `.45`, `.50`, `5.56` and `7.62` is. This is just a lazy low-effort renaming. I can get behind renaming calibres that are identical, like the two (used to be three) types of 9mm ammo, but this? Why?

**Comment by hugmeorelse** [comment]
@KathrinBailey You're being unnecessarily rude. You can say all that without being a dick.

Besides that, I do agree with the points being made here, albeit please say it politely in the future.

@Spc-Dragonfruits I've talked to them further in private, however, I do think what they're saying should be considered. You know how to sprite guns, and you know how to give them a nice aesthetic. Why not do more? You're definitely able to make this an interesting redo, especially if we can modularize this.

**Comment by Draganfrukts** [comment]
I forgot to say that a lot of this stuff was resolved on discord so Yeah .

**Comment by Draganfrukts** [comment]
I need to reopen this in a new branch because i forgot someone fucked with gunsgalore like a couple days ago:]

**Pull request closed without merging**